internal/cephfs/volumeoptions.go

```diff
@@ -230,8 +230,8 @@ func newVolumeOptionsFromVolID(ctx context.Context, volID string, volOpt, secret
 	// before other errors
 	err := vi.DecomposeCSIID(volID)
 	if err != nil {
-		err = fmt.Errorf("error decoding volume ID (%s) (%s)", err, volID)
-		return nil, nil, ErrInvalidVolID{err}
+		return nil, nil, fmt.Errorf("%w: error decoding volume ID (%s) (%s)",
+			ErrInvalidVolID, err, volID)
```

Comment:
This drops the wrapping of `err`. It was probably never used, but doing a `JoinErrors(ErrInvalidVolID, err)` might be better?

Reply:
It wasn't wrapped before, but it's a good idea to do so, so I did. :-)